bin/compile

```diff
@@ -295,6 +295,11 @@ ensureGo() {
 
     export GOROOT="${goPath}"
     PATH="${goPath}/bin:${PATH}"
+
+    # Export GOCACHE if Go >= 1.10
+    if go env | egrep -q '^GOCACHE='; then
+      export GOCACHE="${cache}/${goVersion}/build-cache"
```

Comment:
What is the default and why is that not sufficient?

Reply:
By default `GOCACHE="/root/.cache/go-build"` which isn't carried forward between builds.